database_setup.sql

```diff
@@ -0,0 +1,5 @@
+CREATE DATABASE IF NOT EXISTS groot_credits_service;
+CREATE DATABASE IF NOT EXISTS groot_recruiter_service_dev;
+CREATE DATABASE IF NOT EXISTS groot_meme_service;
+CREATE DATABASE IF NOT EXISTS groot_user_service_dev;
+CREATE DATABASE IF NOT EXISTS groot_caffeine_service_dev;
```

Comment:
Shouldn't quotes be here too? (`groot_quote_service_dev` if I remember correctly)